google/cloud/sql/connector/InstanceConnectionManager.py

```diff
@@ -54,14 +74,31 @@ class InstanceConnectionManager:
     # functionality on Windows. It is recommended to use ProactorEventLoop
     # while developing on Windows.
     _loop: asyncio.AbstractEventLoop = None
+
+    __client_session: aiohttp.ClientSession = None
+
+    @property
+    def _client_session(self) -> aiohttp.ClientSession:
+        if self.__client_session is None:
+            self.__client_session = aiohttp.ClientSession()
+        return self.__client_session
+
+    _credentials: Credentials = None
+
     _instance_connection_string: str = None
+    _instance: str = None
     _project: str = None
     _region: str = None
-    _instance: str = None
-    _credentials: Credentials = None
+
     _priv_key: str = None
     _pub_key: str = None
 
+    _lock: threading.Lock = None
+    _current: concurrent.futures.Future = None
+    _next: concurrent.futures.Future = None
+
+    _delay: int = 15
```

Comment:
This field is only used in one place - can it be inline? Do you really want it to be 15?